src/http-connection/query.codec.ts

```diff
@@ -305,26 +305,43 @@ class QuerySuccessResponseCodec extends QueryResponseCodec {
 
     _decodeTime(value: string): Time<Integer | bigint | number> | LocalTime<Integer | bigint | number> {
         // 12:50:35.556+01:00
+        // 12:50:35+01:00
+        // 12:50:35Z
         const [hourStr, minuteString, secondNanosecondAndOffsetString, offsetMinuteString] = value.split(':')
-        const [secondStr, nanosecondAndOffsetString] = secondNanosecondAndOffsetString.split('.')
-
-        // @ts-expect-error
-        const [nanosecondString, offsetHourString, isPositive]: [string, string, boolean] = nanosecondAndOffsetString.indexOf('+') >= 0 ?
-            [...nanosecondAndOffsetString.split('+'), true] : (
-                nanosecondAndOffsetString.indexOf('-') >= 0 ?
-                    [...nanosecondAndOffsetString.split('-'), false] : (
-                        nanosecondAndOffsetString.indexOf('Z') >= 0 ?
-                            [nanosecondAndOffsetString.slice(0, nanosecondAndOffsetString.length - 1), undefined, true] :
-                            [nanosecondAndOffsetString.slice(0, nanosecondAndOffsetString.length - 1), '0', true]
-                    )
-
-            )
+        let [secondStr, nanosecondAndOffsetString] = secondNanosecondAndOffsetString.split('.')
+        let [nanosecondString, offsetHourString, isPositive, hasOffset]: [string, string, boolean, boolean] = ['0', '0', true, true]
+
+        if (nanosecondAndOffsetString !== undefined) {
+            if ( nanosecondAndOffsetString.indexOf('+') >= 0 ) {
+                [nanosecondString, offsetHourString] = [...nanosecondAndOffsetString.split('+')]
+            } else if (nanosecondAndOffsetString.indexOf('-') >= 0) {
```

Comment:
is there no automated formatting on commit in this repo? I notice there's sometimes a space after the `(` in the if statements, other times not

Reply:
there isn't.